xslt/mathml-to-text/mathml-to-placeholder.xsl

```diff
@@ -33,15 +36,29 @@
     
     <!-- Replace MathML with placeholder -->
     <xsl:template match="m:math[@display eq 'block']">
+        <xsl:variable name="imgsrc" select="@altimg" as="xs:string?" />
+        <xsl:if test="$preserve-mathml-altimg and ($imgsrc)">
+            <img class="mathml-altimg" src="{$imgsrc}" alt="{fnk:translate('placeholder', ., false())}">
```

Comment:
I nordisk standard pleier vi å markere opp bilder som dette (figcaption er vel valgfri men du kan evt. slenge inn `alttext` dit hvis den finnes):

```html
<figure class="image">
    <img alt="image" src="…"/>
    <figcaption>…</figcaption>
</figure>
```

Reply:
Merk at `<figure>` kun er tillatt i blokk-sammenheng da, så i så fall kun gjør dette hvis det er `display="block"`

Reply:
Har allerede tenkt på det, men det var ikke spesifisert - så kunne ikke bare gjette på hvordan det skulle være.

Reply:
Ja, det er jo ikke egentlig av praktisk betydning for lånere (bortsett fra evt. CSS som matcher på klassen).